app/mailers/user_mailer.rb

```diff
@@ -214,7 +214,7 @@ def account_verified(user, email_address, date_time:, sp_name:, disavowal_token:
     return unless email_should_receive_nonessential_notifications?(email_address.email)
 
     with_user_locale(user) do
-      @date = I18n.localize(date_time, format: t('time.formats.event_date'))
+      @date = I18n.l(date_time, format: :event_date)
```

Comment:
removing the call to `t()` was not part of the lint, but it was something I spotted that we could clean up